phantazm-mob/src/main/java/com/github/phantazmnetwork/mob/MobStore.java

```diff
@@ -62,6 +62,11 @@ public void registerMob(@NotNull PhantazmMob mob) {
         uuidToTriggers.put(uuid, createTriggerInstances(mob));
     }
 
+    /**
+     * Gets the {@link PhantazmMob} associated with a {@link UUID}.
+     * @param uuid The {@link UUID} of the {@link PhantazmMob} to get
+     * @return The {@link PhantazmMob} associated with the {@link UUID} or null if none exists
+     */
     public PhantazmMob getMob(@NotNull UUID uuid) {
```

Comment:
definitely would be useful to add a hasMob method to this API, especially since you get an error for trying to register an already-present mob

Reply:
what's the point if getMob returns null

Reply:
`if(getMob(uuid) == null)` is less nice than `if(hasMob(uuid))`